src/ssl.c

```diff
@@ -12565,10 +12457,13 @@ static int CheckcipherList(const char* list)
 static int wolfSSL_parse_cipher_list(WOLFSSL_CTX* ctx, Suites* suites,
         const char* list)
 {
-    int       ret          = 0;
-    int listattribute = 0;
-    char*     buildcipherList = NULL;
-    int tls13Only = 0;
+    int     ret = 0;
+    int     listattribute = 0;
+    int     tls13Only = 0;
+    byte    suitesCpy[WOLFSSL_MAX_SUITE_SZ];
```

Comment:
The `WOLFSSL_MAX_SUITE_SZ` is 300 bytes and will blow out small stack. Please add small stack support for it.

Reply:
Added small stack option.